azerothcore/azerothcore-wotlk
# refactor(Core): replace ACE atomic types with standard C++

Following this https://github.com/cmangos/mangos-wotlk/commit/835e2a02b11a5f4e511461b7ff790190b3f83de3 I removed the ACE atomic type.

## CHANGES PROPOSED:
-  replace ACE atomic type

## ISSUES ADDRESSED:
- related to https://github.com/azerothcore/azerothcore-wotlk/issues/99

## TESTS PERFORMED:
- build and running worldserver

## HOW TO TEST THE CHANGES:
- use the server normally for several days
- ideally, you should use it with players like a normal server for a lot of time

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
